libbpf-rs/src/map.rs

```diff
@@ -26,7 +26,7 @@ impl OpenMap {
         OpenMap { ptr }
     }
 
-    pub fn set_map_ifindex(&mut self, idx: u32) {
+    pub fn set_ifindex(&mut self, idx: u32) {
```

Comment:
Can you comment on why this rename is here, please? Is it intentional? I didn't see it mentioned in the pull request/commit description.

Reply:
The intention of the rename was to make it consistent with the libbpf API `bpf_map__set_ifindex`. I'll edit the commit to mention this - I thought since the change is so minimal it'd make more sense to bundle it in this change to the map setter functions than to have it be its own commit. Though if that's preferred, I dont' mind separating this to its own commit :)

Reply:
Makes sense. Personally I'd split it out just to make it obvious and have basically the description you have at hand, but it's probably fine either way.